charts/cass-operator/templates/configmap.yaml

```diff
@@ -16,11 +18,10 @@ data:
     leaderElection:
       leaderElect: true
       resourceName: b569adb7.cassandra.datastax.com
-{{- if .Values.admissionWebhooks.enabled }}
+{{- if $webhooks }}
```

Comment:
As a user I think it would be counter intuitive to enable webhooks only to see `helm install` or `helm upgrade` disable them. I think it would be better to do one of two things. Either fail if cert-manager isn't installed and `.Values.admissionWebhooks.enabled` is true or make cert-manager a dependency. I would prefer the latter (see #1117.)

Reply:
Since the cert-manager docs explicitly say not to make it a subchart, how about we go with option 1?

Reply:
Validation error sounds fine also, I'll make the change.